ingeniamotion/information.py

```diff
@@ -400,3 +402,14 @@ def get_fw_version(self, alias: str) -> str:
         _, _, fw_version, _ = self.get_drive_info(alias)
         fw_version = fw_version.replace("_", "")
         return fw_version
+
+    def get_vendor_id(self, alias: str) -> str:
```

Comment:
Information class has not to have any function that needs a communication with the drive. I should put this function into Configuration class.